### PR TITLE
Finish docs

### PR DESCRIPTION
**Pull request by ilackarms**
also includes a minor improvement for `wasme list`
BOT NOTES: 
resolves https://github.com/solo-io/wasme/issues/80

### CODE REVIEW

File: docs/content/tutorial_code/push_tutorials/_index.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
```
Comment (ilackarms) [inline docs/content/tutorial_code/push_tutorials/_index.md]: note for reviewer: ignore the left side of this diff, the old file was scrap

**Comment by yuval-k** [comment]
/kick/

File: pkg/cmd/list/list.go [hunk 2]
```diff
@@ -225,15 +225,30 @@ func getPublishedImages(serverAddress, searchQuery string) ([]image, error) {
 	return images, nil
 }
 
-func getTopRepos(serverAddress string) ([]repository, error) {
-	var repos []repository
-	err := getJson(serverAddress, fmt.Sprintf("/api/repositories/top"), &repos)
-	return repos, err
+func getAllRepos(serverAddress string) ([]repository, error) {
+	var projects []project
+	_, err := getJson(serverAddress, fmt.Sprintf("/api/projects"), &projects)
```
Comment (yuval-k) [inline pkg/cmd/list/list.go]: why this change?
